src/main/java/org/folio/rest/support/ResponseUtil.java

```diff
@@ -8,6 +9,18 @@
   private ResponseUtil() {}
 
   public static boolean isUpdateSuccessResponse(Response response) {
-    return response != null && response.getStatus() == HTTP_NO_CONTENT.toInt();
+    return responseIsInStatus(response, HTTP_NO_CONTENT.toInt());
+  }
+
+  public static boolean isDeleteSuccessResponse(Response response) {
+    return responseIsInStatus(response, HTTP_NO_CONTENT.toInt());
+  }
+
+  public static boolean isCreateSuccessResponse(Response response) {
+    return responseIsInStatus(response, HTTP_CREATED.toInt());
+  }
+
+  private static boolean responseIsInStatus(Response response, int expectedStatus) {
```

Comment:
```suggestion
  private static boolean responseHasStatus(Response response, int expectedStatus) {
```

Reply:
I might also be tempted to change the method to take a HttpStatus, then the `.toInt()` can be done in the shared method

Reply:
Make sense, refactored.